drivers/clock_control/CMakeLists.txt

```diff
@@ -5,4 +5,4 @@
 #
 
 zephyr_library()
-zephyr_library_sources(clock_control_ll_nrfxlib.c)
+zephyr_library_sources(clock_control_nrfxlib.c)
```

Comment:
Maybe better to have clock_control_mpsl if the driver is to use this service layer?

Reply:
Yes, but it is also for ble_controller right now

Reply:
Ok. leave as is for now then.